docs/dev_guide/NonContainerizedDeveloperCustomizations.md

```diff
@@ -320,6 +320,40 @@ Example:
 ```
 
 
+## 18. Disable random seeding -
```

Comment:
should we add these new configs to nsfs_config_schema?
and can we add a short test to 
`noobaa-core/src/test/unit_tests/jest_tests/test_nc_nsfs_config_schema_validation.test.js` ?